src/ngraph/op/extractimagepatches.hpp

```diff
@@ -0,0 +1,70 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v3
+        {
+            class NGRAPH_API ExtractImagePatches : public Op
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"ExtractImagePatches", 3};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                ExtractImagePatches() = default;
+                /// \brief Constructs a ExtractImagePatches operation
+                ///
+                /// \param data 4-D Input data to extract image patches
+                /// \param sizes Patch size in the format of [size_rows, size_cols]
+                /// \param strides Patch movement stride in the format of [stride_rows, stride_cols]
+                /// \param rates Element seleciton rate for creating a patch. in the format of
+                /// [rate_rows, rate_cols] \param padding Padding type. it can be any value from
+                /// valid, same_lower, same_upper
+                ExtractImagePatches(const Output<Node>& data,
+                                    const Shape sizes,
+                                    const Strides strides,
+                                    const Shape rates,
```

Comment:
Why not use const Shape&, const Strides&

Reply:
oh, I misunderstood your comment, I will fix it. Thank you.